app/views/content/steps-to-become-a-teacher/_step_1_check_your_qualifications.html.erb

```diff
@@ -2,6 +2,6 @@
 
 <p>You also need a degree in any subject to teach primary and secondary – if you have one or an equivalent qualification, you can do postgraduate teacher training.</p>
 
-<p>If you do not have a degree, you can do undergraduate teacher training to get a degree alongside qualified teacher status (QTS). Or you may be eligible to do a <a href=" https://educationhub.blog.gov.uk/2024/02/04/teacher-degree-apprenticeships-how-they-work-and-when-to-apply/"> teacher degree apprenticeship (TDA)</a> and work in a school, earning a salary while getting a bachelor’s degree and QTS.</p>
+<p>If you do not have a degree, you can do undergraduate teacher training to get a degree alongside qualified teacher status (QTS). Or you may be eligible to do a <a href="/train-to-be-a-teacher/teaching-degree-apprenticeships">teacher degree apprenticeship (TDA)</a> and work in a school, earning a salary while getting a bachelor’s degree and QTS.</p>
```

Comment:
```suggestion
<p>If you do not have a degree, you can do undergraduate teacher training to get a degree alongside qualified teacher status (QTS). Or you may be eligible to do a <a href="/train-to-be-a-teacher/teaching-degree-apprenticeships">teacher degree apprenticeship (TDA)</a>.</p>
```
I would suggest cutting this back a bit - our focus is still postgrad so we don't want to distract too much?